docs/developer.rst

```diff
@@ -147,3 +149,20 @@ As you work on an issue, try adding all the commits into one general commit rath
 Use ``git commit --amend`` to add new changes to a branch.
 
 Use ``-f`` flag to force pushing changes to the branch. ``git push -f origin [branch_name]``
+
+
+Working on the server
+=====================
+
+Data Retriever has a server that can be accessed with permission from the data retriever team at our Gitter channel.
```

Comment:
Data Retriever has a server that can be accessed with permission from the data retriever team.